src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/event_dispatcher_1_event1.txt

```diff
@@ -1,8 +1,8 @@
 <info>[event_dispatcher]</info> Registered listeners for event <info>event1</info>
 
-+-------+-------------------+
-| Order | Callable          |
-+-------+-------------------+
-| #1    | global_function() |
-| #2    | \Closure()        |
-+-------+-------------------+
++-------+-------------------+----------+
+| Order | Callable          | Priority |
++-------+-------------------+----------+
```

Comment:
The command already list the events by order so the information is still available. But finally I think it's more developer friendly to keep it, you don't have to know how Symfony manage order internally :+1: